client/post-editor/post-editor.jsx

```diff
@@ -713,9 +713,6 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
-		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
Alternatively, could we update the reducer for edits to migrate them to the updated post ID when saving a new post (i.e. going from `null` to valid `postId`)?